docs/source/configuration.rst

```diff
@@ -244,6 +246,22 @@ Options reference
      - Time in seconds to wait between two consecutive RPC requests for new events.
        The value applies only to chain NAME, taking precedence over the global poll period.
 
+   * - ::
+
+        min-source-balance = ETH
+
+     - Minimum ETH balance on source chain to fill requests on target chain..
+       The value applies to all chains that don't have the chain-specific min-source-balance defined.
+       Default: ``0.1``.
+
+   * - ::
+
+        [chains.NAME]
+        min-source-balance = ETH
+
+     - Minimum ETH balance on chain ``NAME`` to fill requests originating from it.
```

Comment:
Nitpick: seeing that we refer to `NAME` as `NAME` (without double backticks) elsewhere, I would also remove the backticks here, to be consistent.